hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java

```diff
@@ -280,10 +280,9 @@ public class DatanodeConfiguration extends ReconfigurableConfig {
           " When the difference between volume capacity and used reaches this number," +
           " containers that reside on this volume will be closed and no new containers" +
           " would be allocated on this volume." +
-          " Either of min.free.space or min.free.space.percent should be configured, when both are set then" +
-          " min.free.space will be used."
+          " Max of min.free.space and min.free.space.percent will be configured when both are set."
```

Comment:
will be configured -> will be used as final value

Reply:
done